lib/lrama/state/resolved_conflict.rb

```diff
@@ -8,7 +8,7 @@ class State
     class ResolvedConflict < Struct.new(:symbol, :reduce, :which, :same_prec, keyword_init: true)
       def report_message
         s = symbol.display_name
-        r = reduce.rule.precedence_sym.display_name
+        r = reduce.rule.precedence_sym&.display_name
```

Comment:
It seemed to be nullable. is it acceptable to have nil in the `r` variable?
Currently allows nil to pass type checking.